### PR TITLE
Offline Setup for a variety of patient chart widgets

### PR DESCRIPTION
**Pull request by manuelroemer**
Introduces offline setup functions for a variety of patient chart widgets. Currently restricts functionality to displaying existing patient data. Creating/Editing patient data is out of scope of this PR.

The list of widgets is not yet complete. Currently included:

* patient-banner-app
* patient-biometrics-app
* patient-chart-app
* patient-clinical-view-app
* patient-forms-app
* patient-notes-app
* patient-test-results-app
* patient-vitals-app

### CODE REVIEW

File: packages/esm-patient-banner-app/package.json [hunk 1]
```diff
@@ -52,5 +52,7 @@
     "react-router-dom": "5.x",
     "rxjs": "6.x"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "webpack-stats-plugin": "1.0.3"
```
Comment (FlorianRappl) [inline packages/esm-patient-banner-app/package.json:56]: This should be in the global `devDependencies`. That way only a single change is necessary.

File: packages/esm-patient-biometrics-app/package.json [hunk 2]
```diff
@@ -51,5 +51,7 @@
     "react-router-dom": "5.x",
     "rxjs": "6.x"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "webpack-stats-plugin": "1.0.3"
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/package.json:55]: Same applies to all usages.

File: packages/esm-patient-biometrics-app/webpack.config.js [hunk 3]
```diff
@@ -1,80 +1,85 @@
...
   externals: Object.keys(peerDependencies),
   plugins: [
     new ForkTsCheckerWebpackPlugin(),
     new CleanWebpackPlugin(),
     new BundleAnalyzerPlugin({
-      analyzerMode: env && env.analyze ? "server" : "disabled"
-    })
+      analyzerMode: env && env.analyze ? 'server' : 'disabled',
+    }),
+    new StatsWriterPlugin({
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/webpack.config.js:74]: We could already add this to all webpack configurations.

Comment (FlorianRappl) [reply packages/esm-patient-biometrics-app/webpack.config.js:74]: (that way independent of they provide online or offline components the MFs are prepared)

File: packages/esm-patient-medications-app/webpack.config.js [hunk 4]
```diff
@@ -1,80 +1,84 @@
...
   externals: Object.keys(peerDependencies),
   plugins: [
     new ForkTsCheckerWebpackPlugin(),
     new CleanWebpackPlugin(),
     new BundleAnalyzerPlugin({
-      analyzerMode: env && env.analyze ? "server" : "disabled"
-    })
+      analyzerMode: env && env.analyze ? 'server' : 'disabled',
+    }),
+    new StatsWriterPlugin({
```
Comment (FlorianRappl) [inline packages/esm-patient-medications-app/webpack.config.js:73]: Do we miss the definition of this variable?

File: packages/esm-patient-common-lib/webpack.config.js [hunk 5]
```diff
@@ -1,28 +1,29 @@
-const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
-const { resolve } = require("path");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const { resolve } = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const { StatsWriterPlugin } = require('webpack-stats-plugin');
```
Comment (FlorianRappl) [inline packages/esm-patient-common-lib/webpack.config.js:5]: Sorry my fault - here we don't need it. This is not a MF and is only build for completeness.

Comment (manuelroemer) [reply packages/esm-patient-common-lib/webpack.config.js:5]: Oh okay, will remove it again.

File: packages/esm-patient-medications-app/webpack.config.js (hunk 4 above)
Comment (manuelroemer) [reply packages/esm-patient-medications-app/webpack.config.js:73]: Yes, my fault. I'll run the build before the next commit.

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks! :beers:

**Pull request merged**